### ISSUE TITLE
Poor support for non-English characters in Zilla Slab

### ISSUE THREAD

**Issue by LeoMcA** [mozilla/sumo-project#369]
Zilla Slab has very poor support for diacritics on Latin characters, so in locales [such as Vietnamese](https://support.mozilla.org/vi/get-involved) headings are rendered extremely weirdly since the browser switches to Open Sans on unsupported characters:

![screenshot](https://user-images.githubusercontent.com/12815768/79414329-8a66ab00-7fd4-11ea-9002-8ae9bfcac18e.png)

@kkellydesign is there some sort of CSS feature we can use to make an entire text node fall back to the same font, rather than on a per-character basis? If not I guess we'll have to maintain a blacklist of locales for Zilla Slab.

Also is Open Sans used anywhere else apart from this fallback? Inter also seems to have very good non-English support, so we could use that as the fallback instead.

**Comment by kkellydesign** [comment]
No, not on a per-character basis. I imagine that would end up looking like a ransom note to people who can read the language. A fallback would be easy enough though in CSS. 
```
[lang="fa"] {
--heading-font-family: override stack;
}```

...more or less. It might need to be a little more complicated than that if we want to support IE, but still possible.

**Comment by kkellydesign** [comment]
And no, Open Sans isn't used elsewhere, and it's not installed as a webfont. If Inter or Metropolis will suffice, we should use one of those so we're not installing more.

**Comment by fitojb** [comment]
Neither Inter nor Metropolis support Vietnamese :( (though Inter looks okay-ish when it falls back to Roboto, since it was initially designed over Roboto and the metrics are similar). Different story with Metropolis.

When Mozilla redesigned MDN Web Docs, they already stumbled upon the problem that Zilla Slab has poor language support, and they just blacklisted it in Greek and Cyrillic-script locales. More than a couple of years have passed since then, and one would’ve thought Mozilla would address this, especially since they promised in the blog that introduced Zilla that the font would be expanded. (When they commissioned Fira, it had excellent language support from the beginning.)

**Comment by LeoMcA** [comment]
> No, not on a per-character basis. I imagine that would end up looking like a ransom note to people who can read the language. A fallback would be easy enough though in CSS.

@kkellydesign Sorry, I should have been clearer - a ransom note is exactly what I *didn't* want (and is what happens right now). But perfect, that CSS will work great. Let's not let IE support slow down fixing this for modern browsers.

> Neither Inter nor Metropolis support Vietnamese :( (though Inter looks okay-ish when it falls back to Roboto, since it was initially designed over Roboto and the metrics are similar). Different story with Metropolis.

@fitojb I think Inter must have been updated to support more characters, I did a little test with setting the fallback to Open Sans Light to make it very obvious when that was happening.

Here's `font-family: Inter, "Open Sans Light";`

![Screenshot from 2020-04-17 10-47-14](https://user-images.githubusercontent.com/755354/79556434-fd395a00-8098-11ea-8bd8-0e3aa8792922.png)

And here's `font-family: Metropolis, "Open Sans Light";`

![Screenshot from 2020-04-17 10-46-53](https://user-images.githubusercontent.com/755354/79556555-2b1e9e80-8099-11ea-99e7-e75883418a7d.png)

So I think we should be good to move forward with making Inter the default for headings on diacritic-heavy and non-Latin locales.

**Comment by LeoMcA** [comment]
I've opened a PR to use Inter everywhere in the locales I've found with partial support in Zilla Slab/Metropolis: https://github.com/mozilla/kitsune/pull/4328. I've probably missed some, is anyone aware of a better way to test than just going through the locale picker and clicking around to see if any headings look bad?

@fitojb I haven't included any Greek or Cyrillic-based locales in the blacklist, am I right in thinking that they don't use *any* Latin characters? If so, we can just use the normal CSS font fallback mechanism (and don't need to blacklist them), since we won't run into the situation where certain characters in a word are supported, but the rest aren't. Have I overlooked anything here?
